### PR TITLE
1 project setup

### PR DESCRIPTION
**Pull request by NickKipshidze**
#### Changes
- Provided instructions for running the code on local machine;
- Provided instructions for activating Python virtual environments for multiple operating systems;
- Fixed spelling mistake in `requirements.txt` file;
- Added login/signup routes;
- Added user serializes;
- Updated `.gitignore` file;

Account creation and login has been tested and works correctly;
Django fixture has been tested and works correctly;
Instructions have been corrected;

The `views.py` and `urls.py` code will be refactored in the future.

Everything is working correctly.

### CODE REVIEW

File: twitter/settings.py
Comment (zoiatsitsilashvili) [inline twitter/settings.py:1]: CORS -ის კონფიგურაციისას როგორც მახსოვს ლექციიდან ქვემოთ ესეც მივუთითეთ CORS_ALLOW_ALL_ORIGINS = True

Comment (NickKipshidze) [reply twitter/settings.py:1]: Alright, but in production its not a good idea to allow all origins. That way our API will be vulnerable to CORS attacks. When our frontend will be deployed we can add the origin name to the allowed origins list.

File: initial_data.json
Comment (nodarisvanidze88) [inline initial_data.json:1]: ესესნი იდეაში ჯერ ჯერობით არ გვჭირდება, მაგრამ არც საჭმეს გვთხოვს და არც სასმელს, ამიტომ იყოს ჯერ :)

Comment (NickKipshidze) [reply initial_data.json:1]: Not needed? How are you going to make private messages functionality without the user functionality?

**Review by nodarisvanidze88 — APPROVED** [review]
I think at this moment everything is tip-top, let's move forward

**Pull request merged**